tests/integration/simple-test.js

```diff
@@ -34,5 +34,23 @@ describeComponent(
         done(err)
       })
     })
+    it('renders svg', function (done) {
+      this.render(hbs `<svg id="mySVG"xmlns="http://www.w3.org/2000/svg" viewBox="-350 -250 700 500">
+      <style type="text/css" media="screen">
+        svg { background:#fff; }
+        .face { stroke:#000; stroke-width:20px; stroke-linecap:round }
+      </style>
+      <circle r="200" class="face" fill="red"/>
+      <path fill="none" class="face" transform="translate(-396,-230)" d="M487,282c-15,36-51,62-92,62 c-41,0-77-25-92-61"/>
+      <circle cx="-60" cy="-50" r="20" fill="#000"/>
+      <circle cx="60" cy="-50" r="20" fill="#000"/>
+    </svg>`)
+      capture('svg').then(function (data) {
+        console.log(arguments)
```

Comment:
@sglanzer I'd like to keep that log. Good for a build to see what ResembleJS is returning.